src/wfs/src/test/java/org/geoserver/wfs/v1_1/GetFeatureTest.java

```diff
@@ -637,4 +641,36 @@ public void testFeatureMembers() throws Exception {
         }
     }
 
+    /**
+     * Test GetFeature operation invocations using virtual services end points.
+     * A common issue was XSD caches created only with the types of a certain
+     * workspace and reuse the same cache for other workspaces. The principal
+     * consequence was NULL namespaces in the GetFeature result, e.g null:Buildings.
+     */
+    @Test
+    public void testVirtualServicesInvocation() throws Exception {
+        XpathEngine xpath = XMLUnit.newXpathEngine();
+        // get ride of the current WFS schema
```

Comment:
Typo: "ride" -> "rid"

@nmco your comments and English are great! I would only bother fixing this one if you find yourself updating something more substantial in this file.